packages/rocketchat-livechat/imports/server/rest/facebook.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @api {post} /livechat/facebook Send Facebook message
+ * @apiName Facebook
+ * @apiGroup Livechat
+ *
+ * @apiParam {String} mid Facebook message id
+ * @apiParam {String} page Facebook pages id
+ * @apiParam {String} token Facebook user's token
+ * @apiParam {String} first_name Facebook user's first name
+ * @apiParam {String} last_name Facebook user's last name
+ * @apiParam {String} [text] Facebook message text
+ * @apiParam {String} [attachments] Facebook message attachments
+ */
+RocketChat.API.v1.addRoute('livechat/facebook', {
+	post() {
+		if (!this.bodyParams.text && !this.bodyParams.attachments) {
```

Comment:
Should check if facebook integration is enabled too.

Reply:
done as well =)